core/src/main/java/org/web3j/protocol/Web3j.java

```diff
@@ -14,11 +15,11 @@
     /**
      * Construct a new Web3j instance.
      *
-     * @param web3jService web3j service instance - i.e. HTTP or IPC
+     * @param web3jServices web3j service instances - i.e. HTTP or IPC
      * @return new Web3j instance
      */
-    static Web3j build(Web3jService web3jService) {
-        return new JsonRpc2_0Web3j(web3jService);
+    static Web3j build(Web3jService... web3jServices) {
```

Comment:
Instead of allowing a list of Web3jServices you can extend the Http implementations to accept multiple urls.